### DISCUSSION TITLE
Is the pip install command case-sensitive?

### DISCUSSION THREAD

**Discussion by timbatt** [pallets/flask#4225] [Q&A] [answered]
Wondering if it needs to be `pip install -U Flask` or `pip install -U flask`.  
Just making sure there's no room for typo-squatting.

**Comment by ThiefMaster** [comment]
It's case insensitive. Anything else would be crazy tbh...

**Reply by timbatt** [reply]
Yeah you're right it would be crazy. 
Thanks for the quick answer.

**Comment by davidism** [comment, marked as answer]
Pypi names are case and -_ insensitive.

**Reply by timbatt** [reply]
Ok, thanks for the assistance, sir.
